app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/CreatePasscodeFragment.kt

```diff
@@ -0,0 +1,116 @@
+package io.gnosis.safe.ui.settings.app.passcode
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.inputmethod.EditorInfo
+import android.view.inputmethod.InputMethodManager
+import androidx.core.content.ContextCompat
+import androidx.core.widget.doOnTextChanged
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import io.gnosis.safe.R
+import io.gnosis.safe.ScreenId
+import io.gnosis.safe.databinding.FragmentPasscodeBinding
+import io.gnosis.safe.di.components.ViewComponent
+import io.gnosis.safe.ui.base.SafeOverviewBaseFragment
+import io.gnosis.safe.ui.base.fragment.BaseViewBindingFragment
+import io.gnosis.safe.ui.settings.app.SettingsHandler
+import pm.gnosis.svalinn.common.utils.showKeyboardForView
+import javax.inject.Inject
+
+class CreatePasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBinding>() {
+
+    override fun screenId() = ScreenId.CREATE_PASSCODE
+    private val navArgs by navArgs<CreatePasscodeFragmentArgs>()
+    private val ownerImported by lazy { navArgs.ownerImported }
+
+    @Inject
+    lateinit var settingsHandler: SettingsHandler
+
+    companion object {
+        fun newInstance() = CreatePasscodeFragment()
+    }
+
+    override fun inflateBinding(inflater: LayoutInflater, container: ViewGroup?): FragmentPasscodeBinding =
+        FragmentPasscodeBinding.inflate(inflater, container, false)
+
+    override fun inject(component: ViewComponent) {
+        component.inject(this)
+    }
+
+    override fun onResume() {
+        super.onResume()
+        binding.input.showKeyboardForView()
```

Comment:
this can be done in onViewCreated and we won't need onResume

Reply:
The keyboard is gone when coming back to the app without this.

Reply:
then we can remove same invocation from onViewCreated()